lib/broken_record/slack_notifier.rb

```diff
@@ -1,27 +1,52 @@
 require 'net/http'
 module BrokenRecord
   class SlackNotifier
-    def initialize(options)
-      @options = BrokenRecord::Config.slack_options.merge(options)
+    def initialize(initial_options)
+      @options = default_options
```

Comment:
thoughts on `@options = default_options.merge(initial_options).merge(BrokenRecord::Config.slack_options)`? Three lines of `@options=` is looking a little verbose, but up to you!